setup.py

```diff
@@ -56,8 +56,8 @@
         'pyrouge>=0.1.3',
         'sacrebleu~=1.0',
         'requests~=2.22',
-        'transformers==2.11',
-        'sentencepiece>=0.1.83,<0.2.0',
+        'transformers==3.5.1',
+        'sentencepiece==0.1.91',
```

Comment:
Do we really need these specific versions, or can we relax the requirements a bit?

Reply:
huggingface pins sentencepiece to 0.1.91.  Not sure if it works with other versions...

Reply:
But we don't need that specific version, so you should leave our requirement more open and then pip will pick up the more stringent requirement from huggingface.